libs/tup-components/src/tickets/TicketDetailModal/TicketModal.global.css

```diff
@@ -133,3 +134,28 @@
   padding-right: 5rem;
   margin-left: 1rem;
 }
+
+.status-wrapper {
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 2rem;
+}
+
+.status-checkbox {
+  display: flex;
+  align-items: center;
+  margin-bottom: 0.75em;
+}
+
+.status-checkbox > input {
+  margin-right: 0.5em;
+}
```

Comment:
**Prefer Core-Styles.** Good job. I noticed too late that Core-Styles offers solutions with [`c-form`](https://tacc.utexas.edu/static/ui/components/preview/c-form--portal.html). Please review:
- #405

Reply:
Also, your PR could go in as is. My PR can be reviewed after yours is good and merged. Either way.

Reply:
I resolved merge conflicts in #405.